bodhi/server/bugs.py

```diff
@@ -201,15 +201,17 @@ def close(self, bug_id, versions, comment):
                 fixedin = [v.strip() for v in bug.fixed_in.split()]
                 # Strip out any empty strings (already stripped)
                 fixedin = [v for v in fixedin if v]
-                # And add our build if its not already there
-                if version not in fixedin:
-                    fixedin.append(version)
 
                 # There are Red Hat preferences to how this field should be
                 # structured.  We should use:
                 # - the full NVR as it appears in koji
                 # - space-separated if there's more than one.
-                args['fixedin'] = " ".join(fixedin)
+                fixedinString = " ".join(fixedin)
```

Comment:
For [PEP-8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), this should be named ```fixedin_str``` (or ```fixedin_string``` if you prefer the full name).